guides/common/modules/proc_applying-errata-to-multiple-hosts.adoc

```diff
@@ -1,7 +1,7 @@
 [id="Applying_Errata_to_Multiple_Hosts_{context}"]
 = Applying Errata to Multiple Hosts
 
-Use these procedures to review and apply errata to multiple RHEL 7 hosts.
+Use these procedures to review and apply errata to multiple RHEL 8 hosts.
```

Comment:
I lean towards just dropping the version number here. This procedure applies to any host managed and could be RHEL 6,7,8,9.